### PR TITLE
[iam] Replace subject with commmon name in CertHandler::CreateKey

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: include/aos/iam/certhandler.hpp [hunk 1]
```diff
@@ -68,13 +68,13 @@ class CertHandler {
      * Creates key pair.
      *
      * @param certType certificate type.
-     * @param subject subject of CSR.
+     * @param subject common name of the subject.
```
Comment (al1img) [inline include/aos/iam/certhandler.hpp:71]: Add proper param name: `subjectCommonName`

File: include/aos/iam/modules/certmodule.hpp [hunk 2]
```diff
@@ -191,12 +191,12 @@ class CertModule {
     /**
      * Creates certificate request.
      *
-     * @param subject certificate subject.
+     * @param subjectCommonName common name of the subject.
```
Comment (al1img) [inline include/aos/iam/modules/certmodule.hpp:194]: ditto

File: include/aos/iam/certhandler.hpp (hunk 1 above)
Comment (mykola-kobets-epam) [reply include/aos/iam/certhandler.hpp]: fixed

File: include/aos/iam/modules/certmodule.hpp (hunk 2 above)
Comment (mykola-kobets-epam) [reply include/aos/iam/modules/certmodule.hpp:194]: param name is already correct

**Review by al1img — APPROVED** [review]

**Pull request merged**